canonical-websites/www.ubuntu.com
# Create new appstore page

## Done

* Created new /internet-of-things/appstore page
* Added a direct from /iot-appstore

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [new page](http://0.0.0.0:8001/internet-of-things/appstore)
- compare it to the [copy doc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit#)

## Issue / Card

Fixes #2492
